controllers/src/main/scala/cromwell/pipeline/controller/AuthController.scala

```diff
@@ -31,16 +32,22 @@ class AuthController(authService: AuthService)(implicit executionContext: Execut
       },
       path("signUp") {
         post {
-          entity(as[SignUpRequest]) { request =>
-            FormValidatorNel.validateForm(request) match {
-              case Valid(validatedRequest) =>
-                onComplete(authService.signUp(validatedRequest)) {
-                  case Success(Some(authResponse)) => setSuccessAuthRoute(authResponse)
-                  case _                           => complete(StatusCodes.BadRequest)
-                }
-              case Invalid(errors) =>
-                complete(StatusCodes.BadRequest -> errors.toList.map(_.toMap))
-            }
+          entity(as[SignUpRequest]) {
+            request =>
+              FormValidatorNel.validateForm(request) match {
+                case Valid(validatedRequest) =>
+                  onComplete(authService.signUp(validatedRequest)) {
+                    case Success(Some(authResponse)) => setSuccessAuthRoute(authResponse)
+                    case _                           => complete(StatusCodes.BadRequest)
+                  }
+                case Invalid(errors) =>
+                  complete(
+                    StatusCodes.BadRequest -> PasswordProblemsResponse(
+                      value = request.password,
+                      errors = errors.toList.map(_.toMap)
+                    )
+                  )
+              }
```

Comment:
Why we wrap in `PasswordProblemsResponse` in `Invalid` case?   

As far as I can see, we're using `FormValidatorNel.validateForm` which validates not only password, but other parts of the entity as well.